pkg/networkmanager/interfaces.js

```diff
@@ -1445,6 +1445,12 @@ PageNetworking.prototype = {
         $("#networking-add-bridge").click($.proxy(this, "add_bridge"));
         $("#networking-add-vlan").click($.proxy(this, "add_vlan"));
 
+        $("#networking-add-team").hide();
+        cockpit.script("which rpmquery && rpmquery NetworkManager-team", { err: "ignore" }).
```

Comment:
> This is a HACK ... needs an upstream bug and hope of a real fix there.

What would the real fix be in your opinion?  Support in the NM API to list plugins?